openmc_plotter/plotgui.py

```diff
@@ -19,7 +19,7 @@
 from .scientific_spin_box import ScientificDoubleSpinBox
 from .custom_widgets import HorizontalLine
 
-if is_pyqt5():
+if QtCore.qVersion():
     from matplotlib.backends.backend_qt5agg import FigureCanvas
 else:
     from matplotlib.backends.backend_qt5agg import FigureCanvas
```

Comment:
Looking at this again, we can probably just remove this check. The lines in under the `if` and `else` are exactly the same...

```suggestion
from matplotlib.backends.backend_qt5agg import FigureCanvas
```